src/main/webapp/search-style.css

```diff
@@ -26,7 +26,7 @@ textarea, select, fieldset {
   width: 50%;
 }
 
-#search-button, #logout-button {
+#search-button, #logout-button, #private-logout-button, #add-to-search, #search-to-add, #add-button {
```

Comment:
Might think about extracting this to a class name in the future if it further expands.

Reply:
Made all of this into the class course-button